pkg/config/vsphereuser.go

```diff
@@ -0,0 +1,82 @@
+package config
+
+import (
+	_ "embed"
+	"os"
+)
+
+const (
+	EksavSphereUsernameKey = "EKSA_VSPHERE_USERNAME"
+	EksavSpherePasswordKey = "EKSA_VSPHERE_PASSWORD"
+	// Username and password for cloud provider
+	EksavSphereCPUsernameKey = "EKSA_VSPHERE_CP_USERNAME"
+	EksavSphereCPPasswordKey = "EKSA_VSPHERE_CP_PASSWORD"
+	// Username and password for the CSI driver
+	EksavSphereCSIUsernameKey = "EKSA_VSPHERE_CSI_USERNAME"
+	EksavSphereCSIPasswordKey = "EKSA_VSPHERE_CSI_PASSWORD"
+)
+
+type VSphereUserConfig struct {
+	EksaVsphereUsername    string
+	EksaVspherePassword    string
+	EksaVsphereCPUsername  string
+	EksaVsphereCPPassword  string
+	EksaVsphereCSIUsername string
+	EksaVsphereCSIPassword string
+}
+
+//go:embed static/globalPrivs.json
+var VSphereGlobalPrivsFile string
```

Comment:
I actually did that originally, but it meant I didn't get linting errors if a particular file wasn't present. I'm kind of paranoid about forgetting to bundle static assets in builds because python makes it suuuper easy to shoot yourself in the foot this way when creating packages.

Agreed it's not great to need to add each new file here.